CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 ## 1.2
 
+### 1.2.1 - 2024-03-29
+
+([full changelog](https://github.com/jupyterhub/nbgitpuller/compare/1.2.0...1.2.1))
+
```

Comment:
Based on #346, 1.2.1 is needed if you're using the newest version of jupyterhub, right? If so, I think we should manually mention it here. Otherwise LGTM

Reply:
Yes, 1.2.1 fixed compatibility with JupyterHub >=4.1

Reply:
Great point @yuvipanda - added a note about it. Going for a release!